src/moveFile.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+'use strict';
+
+const { existsSync } = require('fs');
+const fs = require('fs/promises');
+const path = require('node:path');
+
+const doesNotExistError = (pathName) => {
+  console.error(`Error. Path ${pathName} does not exist.`);
+};
+
+const moveFile = async(input) => {
+  if (input.length !== 4) {
+    console.error('Please, enter exact two paths.');
```

Comment:
....aaaand user stuck :) you don't give any chance for mistake😅😂 Maybe we can give user one more attempt? For usability

Reply:
sorry, i dont really understand where the problem here, the app just closes and user can enter the commands again, or what do you mean when you said give user one more attempt?